convert_to_dir.go

```diff
@@ -0,0 +1,141 @@
+package bindata
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+// translateToDir generates splited file
+func translateToDir(c *Config, toc []Asset, wd string) error {
+	if err := generateCommonFile(c, toc); err != nil {
+		return err
+	}
+
+	for i := range toc {
+		if err := generateOneAsset(c, &toc[i], wd); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func generateCommonFile(c *Config, toc []Asset) error {
+	// Create output file.
+	fd, err := os.Create(filepath.Join(c.Output, "common.go"))
```

Comment:
I prefer to use `bindata.go` as the common file, to minimize conflict if user want to generate to directory that has the `common.go` (since `-split` now assume output as dir, user can not change the output file name.)

Reply:
ok, u're right, it's better.